### PR TITLE
Add info on Mathfest talks to main page

### PR DESCRIPTION
**Pull request by siwelwerd**

### CODE REVIEW

File: site/description.html [hunk 1]
```diff
@@ -1,3 +1,10 @@
+<p class="notice">
+    Are you at Mathfest this week?  Join us on Friday 8/9 for these four TBIL-themed talks!
+    - "Implementations of Team-Based Inquiry Learning Across Instructional Contexts", 8:40 AM, Room 314.  
+    - "Team-Based Inquiry Learning in College Algebra and Precalculus", 9:20 AM, Room 311. 
+    - "Tailoring TBIL Resources Using Github", 4:40 PM, Room 313. 
+    - "Team-Based Inquiry Learning in Linear Algebra", 5:20 PM, Room 314.
```
Comment (siwelwerd) [inline site/description.html:6]: Oh, I guess this is HTML but the others are markdown...
```suggestion
<ul>
    <li> "Implementations of Team-Based Inquiry Learning Across Instructional Contexts", 8:40 AM, Room 314. </li>
    <li> "Team-Based Inquiry Learning in College Algebra and Precalculus", 9:20 AM, Room 311. </li>
    <li>"Tailoring TBIL Resources Using Github", 4:40 PM, Room 313. </li>
    <li>"Team-Based Inquiry Learning in Linear Algebra", 5:20 PM, Room 314. </li>
</ul>
```

**Comment by siwelwerd** [comment]
@StevenClontz I am doing something stupid and cannot get the little advertisement about Mathfest to show up  at the top of the main page like I wanted to.  Can you take a look?

**Comment by StevenClontz** [comment]
This has something to do with being branched off a stale copy of main, maybe? Hmm..

**Comment by StevenClontz** [comment]
So I think I learned something today: our CI/CD seems to build based upon what would happen after merging into `main`, which would explain why something might look different for a contributor locally if their branch wasn't up to date with main. This is a good thing, but something we should be aware of.

**Pull request merged**